demo/src/app/components/+timepicker/demos/arrowkeys/arrowkeys.html

```diff
@@ -0,0 +1,5 @@
+<p>Arrow keys disabled</p>
```

Comment:
IMO it's better for each of these demos to have a button that disables/enables arrow keys or mosuewheel